processing-service-core/api/swagger/swagger.yaml

```diff
@@ -952,7 +954,123 @@ paths:
           description: The specified Workflow could not be found.
           schema:
             $ref: "#/definitions/errorMessage"    
+  ##########################
+  ##  Default Config Store
+  ##########################
+  /globalConfig/{key}:
+    x-swagger-router-controller: globalConfig
+    parameters:
+      - name: key
+        in: path
+        type: string
+        description: The identifier of global config for tenant.
```

Comment:
This isn't related to any tenant. Should read as 'The unique identifier for the global configuration'.